.build/workspace-state.json

```diff
@@ -0,0 +1 @@
+{"object": {"artifacts": [], "dependencies": []}, "version": 4}
```

Comment:
This should be gitignored. Would you mind adding the following to the `.gitignore`? (This is the SPM-default .gitignore you get if you run `swift package init`)

```
.DS_Store
/.build
/Packages
/*.xcodeproj
xcuserdata/
```